State/Type

```diff
@@ -1,2 +1 @@
-
-λ(s : Type) → λ(a : Type) → s → { state : s, val : a }
+λ(s : Type) → ../StateT/Type  s ../Id/Type 
```

Comment:
And, BTW, this one I like using `StateT` (unlike `Reader/Type`) because it has additional structure (field names) that we’d prefer not to duplicate, so it doesn’t get out of sync when one of the files is edited later.